app/src/main/scala/org/alephium/explorer/persistence/queries/HashrateQueries.scala

```diff
@@ -16,9 +16,10 @@
 
 package org.alephium.explorer.persistence.queries
 
-import java.time._
+import scala.concurrent.ExecutionContext
```

Comment:
Why `ExecutionContext` was not needed before?

Reply:
to do the `map` here https://github.com/alephium/explorer-backend/blob/7cbf417d4e9408536fbb3a0ece59feb905b82208/app/src/main/scala/org/alephium/explorer/persistence/queries/HashrateQueries.scala#L113 but I realize those functions using that need the EC are just for testing purpose, I'll clean that, nice catch

Reply:
let's keep it like this for now, this will be anyway fixed with https://github.com/alephium/explorer-backend/issues/154